src/components/Channel/emojiData.ts

```diff
@@ -1,20 +1,22 @@
-export const emojiSetDef = {
+import type { Data as EmojiData } from 'emoji-mart';
+
+import type { CommonEmoji, EmojiSetDef, MinimalEmoji } from '../../context';
```

Comment:
this is with the intention that we get rid of a lot of these BS index.js/ts files that make VSCode command searching for files a little annoying
```suggestion
import type { CommonEmoji, EmojiSetDef, MinimalEmoji } from '../../context/ChannelContext';
```